packages/react-components/src/lib/cesium-map/data-sources/drawings.data-source.tsx

```diff
@@ -55,79 +57,90 @@ export const CesiumDrawingsDataSource: React.FC<RCesiumDrawingDataSourceProps> =
+      if (drawState.drawing) {
+        switch (drawState.type) {
+          case DrawType.POLYGON:
+            // eslint-disable-next-line
+            (drawHelper as any).startDrawingPolygon({
+              callback: (positions: PrimitiveCoordinates) => {
+                //// MAKE polygon editable example
+                // var polygon = new DrawHelper.PolygonPrimitive({
+                //   positions: positions,
+                //   width: 5,
+                //   geodesic: true
+                // });
+                // mapViewer.scene.primitives.add(polygon);
+                // polygon.setStrokeStyle(Color.AQUA,1);
+                // polygon.setEditable();
+                // polygon.addListener('onEdited', function(event) {
+                //   console.log('Polygone edited, ' + event.positions.length + ' points');
+                // });
+                drawState.handler({
+                  primitive: positions,
+                  type: DrawType.POLYGON,
+                  geojson: polygonToGeoJSON(positions as Cartesian3[]),
+                });
+              },
+            });
+            break;
+          case DrawType.BOX:
+            // eslint-disable-next-line
+            (drawHelper as any).startDrawingExtent({
+              callback: (positions: PrimitiveCoordinates) => {
+                //// MAKE box editable example
+                // var extentPrimitive = new DrawHelper.ExtentPrimitive({
+                //   extent: positions,
+                //   material: Cesium.Material.fromType(Cesium.Material.StripeType)
+                // });
+                // mapViewer.scene.primitives.add(extentPrimitive);
+                // extentPrimitive.setStrokeStyle(Color.AQUA,1);
+                // extentPrimitive.setEditable();
+                // extentPrimitive.addListener('onEdited', function(event) {
+                //   console.log('Extent edited: extent is (N: ' + event.extent.north.toFixed(3) + ', E: ' + event.extent.east.toFixed(3) + ', S: ' + event.extent.south.toFixed(3) + ', W: ' + event.extent.west.toFixed(3) + ')');
+                // });
+                drawState.handler({
+                  primitive: positions,
+                  type: DrawType.BOX,
+                  geojson: rectangleToGeoJSON(positions as Rectangle),
+                });
+              },
+            });
+            break;
+          default:
+            break;
+        }
+      } else {
+        // eslint-disable-next-line
+        (drawHelper as any).stopDrawing();
       }
     }
   }, [drawState, drawHelper]);
 
   const renderGraphicsComponent = (
     drawEntity: IDrawing
   ): React.ReactElement => {
+    const coordinates =
+      // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
+      drawEntity.coordinates !== undefined
+        ? drawEntity.coordinates
+        : geoJSONToPrimitive(
+            drawEntity.type,
+            drawEntity.geojson as FeatureCollection
+          );
```

Comment:
why suppressing `@typescript-eslint/no-unnecessary-condition` ? is it an unnecessary condition?

Reply:
Local linting problems, 
"Eagle eye", DONE